website/docs/docs/cloud/git/setup-azure.md

```diff
@@ -373,6 +373,17 @@ A dbt Cloud account admin with access to the service user's Azure DevOps account
 
 Once connected, dbt Cloud displays the email address of the service user so you know which user's permissions are enabling headless actions in deployment environments. To change which account is connected, disconnect the profile in dbt Cloud, sign into the alternative Azure DevOps service account, and re-link the account in dbt Cloud.
 
+<Expandable alt_header="I can't select the service user, dbt Cloud automatically connects my personal user" > 
```

Comment:
not a blocker but i recommend changing this to an h3 or h4 instead of expandable, faq type of heading. For example: 

### Using Azure SD for SSO with dbt Cloud and Microsoft tools